### PR TITLE
Documents operator-lib PR #100.

### PR DESCRIPTION
**Pull request by fgiloux**
**Description of the change:**

The log field could not be set as it was unexported. The log strategy also needed to be aligned with the rest of the library.
[operator-lib PR #100](https://github.com/operator-framework/operator-lib/pull/100)

**Motivation for the change:**

It was not usable as it was and the documentation needs to reflect the implementation change

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [x] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by fgiloux** [comment]
cc @camilamacedo86

**Comment by jberkhahn** [comment]
looks like you forgot to sign your commit

**Comment by fgiloux** [comment]
> looks like you forgot to sign your commit

Done

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

File: website/content/en/docs/best-practices/resource-pruning.md [hunk 1]
```diff
@@ -106,10 +104,10 @@ from the cluster.
 
 Here is an example of a *preDelete* hook:
 ```golang
-func myhook(cfg Config, x ResourceInfo) error {
-	fmt.Println("myhook is called ")
-       	if x.GVK.Kind == PodKind {
-                req := cfg.Clientset.CoreV1().Pods(x.Namespace).GetLogs(x.Name, &v1.PodLogOptions{})
+func myhook(cfg Config, res ResourceInfo) error {
+        log.V(4).Info("pre-deletion", "GVK", res.GVK, "namespace", res.Namespace, "name", res.Name)
```
[the remark this answers is not in the document]
Comment (fgiloux) [reply website/content/en/docs/best-practices/resource-pruning.md:108]: @camilamacedo86 as soon as the code fix has been merged I will update this doc PR. There are a couple of places that would need to be changed to align to the current version of the code fix and this is one of them.

**Comment by fgiloux** [comment]
Doc updated to align with the modifications of the code PR

**Comment by camilamacedo86** [comment]
I think it address all comments/suggestions made so far
So, we are ok to move forward. 
Anyway, please feel free to raise issues or contact us if you see any hall for improvements.

**Pull request merged**